### PR TITLE
HDDS-11477. [doc] Add configuration description for datanode docs

### PR DESCRIPTION
**Pull request by aierate**
## What changes were proposed in this pull request?
Due to the lack of configuration related descriptions on the Datanode docs new learners are unable to deploy the Datanode service quickly. Therefore, it is necessary to add some configuration descriptions of Datanodes to provide guidance for new learners.
In this patch, we've added configuration details to the DataNode documentation.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11477

## How was this patch tested?
The test passed using the Hugo tool.
image

### CODE REVIEW

File: hadoop-hdds/docs/content/concept/Datanodes.md [hunk 1]
```diff
@@ -76,3 +76,15 @@ blocks that get reported. That is a 40x reduction in the block reports.
...
+## Notable configurations
+
+key | default | <div style="width: 300px;">description</div>
+----|---------|------------
+dfs.container.ratis.datanode.storage.dir | none | This directory is used for storing Ratis metadata like logs.
+ozone.scm.datanode.id.dir | none | The path that datanodes will use to store the datanode ID.
+hdds.datanode.dir | none | Determines where on the local filesystem HDDS data will be stored.
+hdds.datanode.dir.du.reserved | none | Reserved space in bytes per volume. Always leave this much space free for non dfs use.
+ozone.metadata.dirs | none | Directory to store persisted data (RocksDB).
+ozone.recon.address | 0.0.0.0:9891 | RPC address of the Recon.
```
Comment (whbing) [inline hadoop-hdds/docs/content/concept/Datanodes.md:90]: `0.0.0.0:9891 ` is recon server side config. Can add a description: "RPC address of the Recon.  Use \<host:port\> to connect Recon"

**Review by whbing — COMMENTED** [review]
@aierate Thanks for working on this ! Some comments left.

**Comment by whbing** [comment]
Could you also improve the `Datanodes.zh.md`?

**Comment by aierate** [comment]
> Could you also improve the `Datanodes.zh.md`?

@whbing .Thank you for your review. I have modified the description of Recon and added a Chinese description. Please help me review again here. Thanks.

File: hadoop-hdds/docs/content/concept/Datanodes.zh.md [hunk 2]
```diff
@@ -49,3 +49,16 @@ Ozone 的存储容器是一个自包含的超级块，容器中包含一系列
 SCM 如何获得容器的位置？这一点和现有的 HDFS 十分相似。数据节点会定期发送类似于块报告的容器报告，容器报告比块报告的内容简洁的多，比如，对于一个存储容量为 196 TB 的集群，Ozone 大概会拥有四万个容器，相比于 HDFS 的一百五十万个块，块报告数量缩减为四十分之一。
 
 这种间接管理的方式大大地提高了 Ozone 的扩展性，因为 SCM 需要处理的块数据大大减少，且命名服务（OM）作为一个独特的服务主体对于扩展 Ozone 具有重要意义。
+
+
+
```
Comment (whbing) [inline hadoop-hdds/docs/content/concept/Datanodes.zh.md:54]: Two blank lines are enough.

File: hadoop-hdds/docs/content/concept/Datanodes.zh.md [hunk 3]
```diff
@@ -49,3 +49,16 @@ Ozone 的存储容器是一个自包含的超级块，容器中包含一系列
...
+## 需要关注的配置项
+
+配置项 |默认值 | <div style="width:300px;">描述</div>
+----|---------|------------
+dfs.container.ratis.datanode.storage.dir | none | 该目录用于存储 Ratis 元数据，如日志。
+ozone.scm.datanode.id.dir | none | 数据节点上用于存储数据节点 ID 的路径。 
+hdds.datanode.dir | none | 此配置决定数据节点上的数据将存储在本地文件系统的哪个位置。
+hdds.datanode.dir.du.reserved | none | 每个卷保留的存储空间（以字节为单位）。始终为非DFS用途保留这么多空闲空间。
+ozone.metadata.dirs | none | 用于存储持久化数据（RocksDB）的目录。
+ozone.recon.address | 0.0.0.0:9891 | Recon的RPC地址. 使用 <host:port> 连接到Recon.
```
Comment (whbing) [inline hadoop-hdds/docs/content/concept/Datanodes.zh.md:64]: .  --> 。

Comment (aierate) [reply hadoop-hdds/docs/content/concept/Datanodes.zh.md]: done.

**Review by whbing — APPROVED** [review]

File: hadoop-hdds/docs/content/concept/Datanodes.md [hunk 4]
```diff
@@ -76,3 +76,15 @@ blocks that get reported. That is a 40x reduction in the block reports.
...
 different service are critical to scaling Ozone.
+
+
+## Notable configurations
+
+key | default | <div style="width: 300px;">description</div>
+----|---------|------------
+dfs.container.ratis.datanode.storage.dir | none | This directory is used for storing Ratis metadata like logs.
+ozone.scm.datanode.id.dir | none | The path that datanodes will use to store the datanode ID.
+hdds.datanode.dir | none | Determines where on the local filesystem HDDS data will be stored.
```
Comment (Tejaskriya) [inline hadoop-hdds/docs/content/concept/Datanodes.md:87]: Minor suggestion, as the sentence feels slightly confusing
```suggestion
hdds.datanode.dir | none | Determines where HDDS data will be stored on the local filesystem.
```

**Review by Tejaskriya — COMMENTED** [review]
Thanks for the patch, Looks good to me over all. Just a small suggestion below

**Review by Tejaskriya — APPROVED** [review]
Thanks for addressing it, LGTM!

**Comment by whbing** [comment]
Merged. Thanks @Tejaskriya for review , and  @aierate for the pr !

**Pull request merged**